spec/record_spec.rb

```diff
@@ -0,0 +1,153 @@
+require_relative 'spec_helper'
+
+describe CodaStandard::Record do
+  before :each do
+    @header_record = CodaStandard::Record.new("0000031031520005                  MMIF SA/BANK              GEBABEBB   00538839354 00000                                       2")
+    @old_balance_data_record = CodaStandard::Record.new("10016539007547034 EUR0BE                  0000000057900000300315MMIF SA/EVOCURE                                              017")
+    @data_movement1_record = CodaStandard::Record.new("21000100000001500000103        0000000000500000010415001500001101100000834941                                      31031501601 0")
+    @data_movement1b_record = CodaStandard::Record.new("21000100000001500000103        0000000000500000010415001500001001100000834941                                      31031501601 0")
+    @data_movement2_record = CodaStandard::Record.new("2200010000                                                                                        GKCCBEBB                   1 0")
+    @data_movement3_record = CodaStandard::Record.new("2300010000BE53900754703405                  EURLASTNM PERSON                                                                 0 1")
+    @data_information2_record = CodaStandard::Record.new("32000200015 STREET                                     3654 CITY BELGIQUE                                                    0 0")
```

Comment:
We should use a 'let' to assign these variables, outside of the before block. It looks prettier.